# Problem Set 5

Due: October 18, 11:59 PM.

## Assignment

Answer the following questions related to performing the requested tasks and then
include your answers in an electronic file, save as a single PDF. Upload your PDF file
to the Blackboard Assignment page for Problem Set 5 by 11:59 PM on the due date.

### Part 0

Develop a CV for yourself—if you already have one, take some time to refine it.  Include the information we’ve talked about in class and anything else you think might be relevant.  Be sure to format it is an attractive way so you win the award/grant/job!!!

### Part 1

Identify and list 5 internship/job opportunities in bioinformatics that might be of interest to you.  Give the details of the opportunity and explain why these opportunities are of interest to you.

### Part 2

Following the instructions below, taking care to answer all questions in the assignment. Show your work where applicable.

1.	Identify a protein of interest and find an amino acid sequence for this protein (make sure it is somewhere between 50 and 500 amino acids long).  What is your protein?  Why did you choose it?
2.	Use an appropriate BLAST approach to identify at least 4 other amino acid sequences related, but not identical to your protein sequence. What BLAST approach did you use?  Why?  What parameter values did you use?
3.	Perform a multiple sequence alignment on your selected amino acid sequences.  What alignment algorithm did you use?  Why?  What parmeter values did you use?  Show your MSA.
4.	Using your MSA, identify secondary structural elements along the amino acid sequence.  How did you identify these structural elements (what software did you use?)?  How does the software actually make the inferences of secondary structure?  What assumptions are you making in this analysis?  What structural elements did you predict from your data?  Do your results make sense, why or why not?  Show the output of the structural analysis.
5.	Identify tertiary structural elements in your alignment. How did you identify these structural elements (what software did you use?)?  How does the software actually make the inferences of tertiary structure?  What assumptions are you making in this analysis?  What structural elements did you predict from your data?  Do your results make sense, why or why not?  Show the output of the structural analysis.
6.	 Find a 3-D model that is close to your protein of interest and map key variants in your alignment on the 3-D structure.  Show your result.  Isn’t that the coolest thing ever?!

