# Lecture 3 in class work

Using the ‘Lecture3Data.fasta’ file, identify the sequences in the file.  Rename them by species and gene, e.g., Genus_species_gene – for our ApoE example, Homo_sapiens_ApoE.  

Work in a group of at least two individuals.  One of you convert the nucleotides to amino acids.  The other use the nucleotides.  Now …

1. Align the sequences (either amino acids or nucleotides) to each other using MAFFT https://mafft.cbrc.jp/alignment/software/.

2. What does your alignment tell you about your sequences?  Click on the ‘phylogeny’ or ‘tree’ view.  Identify paralogs and orthologs.

3. Examine the parameter options for the alignment.  What do these mean?  We’ll discuss coming up!

